README.md

```diff
@@ -54,32 +60,73 @@ $ git clone https://github.com/IBM/casis-satellite-tracker
+![](doc/source/images/WatsonAssistantCredentials.png)
 
-### 4. Import the Conversation workspace
+### 3. Import the Watson Assistant workspace
 
-Launch the **Watson Conversation** tool. Use the **import** icon button on the right
+Once you have created your instance of Watson Assistant, click `Launch Tool`, then click the `Workspaces` tab. Import the workspace by clicking the upload icon:
 
-Find the local version of [`data/workspaces/satelliteChat.json`](data/workspaces/satelliteChat.json) and select
-**Import**. Find the **Workspace ID** by clicking on the context menu of the new
-workspace and select **View details**. Save this ID for later.
+![](doc/source/images/UploadWorkspaceJson.png)
 
-*Optionally*, to view the conversation dialog select the workspace and choose the
-**Dialog** tab, here's a snippet of the dialog:
+Click `Choose a file` and navigate to [`data/workspaces/SatTrackWorkspace.json`](data/workspaces/SatTrackWorkspace.json) in this repo. Click `Import`.
 
-![](doc/source/images/dialog.PNG)
+Get the Workspace ID by clicking the 3 vertical dots on the `Workspaces` tab. Save this for later.
 
-### 5. Configure credentials
+<p align="center">
+  <img width="200" height="300" src="doc/source/images/GetAssistantDetails.png">
+</p>
+
+### 4. Get the Watson Assistant credentials
 
 The credentials for IBM Cloud Watson Assistant service can be found
-by selecting the ``Service Credentials`` option for each service.
+by selecting the ``Service Credentials`` option for the service. You
+saved these in [step #2](#2-create-watson-services-with-ibm-cloud).
+
+The `WORKSPACE_ID` for the Watson Assistant workspace was saved in
+[step #3](#3-import-the-watson-assistant-workspace).
+
+### 5. Create a Node-RED Workspace
+
+From the the [IBM Cloud Catalog](https://console.bluemix.net/catalog/) navigate to `Platform` -> `Boilerplates` and choose [Node-RED Starter](https://console.bluemix.net/catalog/starters/node-red-starter). Choose a name and click `Create`.
+
+Once the App has deployed, click on `Visit App URL`
+
+![](doc/source/images/NodeRedVisitAppURL.png)
+
+Follow the instructions to `Secure your Node-RED editor` and `Browse available IBM Cloud nodes`. Click `Finish` and then click `Go to your Node-RED flow editor`.
+
+### 6 Install additional nodes
+
+You will need to install the following additional nodes:
+
+```
+[node-red-contrib-credentials](https://flows.nodered.org/node/node-red-contrib-credentials)
+[node-red-contrib-web-worldmap](https://flows.nodered.org/node/node-red-contrib-web-worldmap)
+[node-red-contrib-satellites](https://flows.nodered.org/node/node-red-contrib-satellites)
+
+```
+
+### 6.a Build the Node-RED flow manually
+
+### 6.b Import the completed Flow
+
+We will walk through the steps to build the Node-RED flow, but you can import the completed Flow. Copy the flow to your machine's clipboard by navigating to `data/Node/`.
+A flow can be moved to a Mac OS clipboard with:
```

Comment:
put this on a new line and use triple backticks for the copy commands (same for each OS)

Reply:
Done. Mark had complained that a single backtick would work for one-liners, but I like the triple backtick as a standard. WE just need to converge on one solution...